owncloudData/src/test/java/com/owncloud/android/data/files/repository/OCFileRepositoryTest.kt

```diff
@@ -198,6 +199,32 @@ class OCFileRepositoryTest {
         }
     }
 
+    @Test
+    fun `getDownloadedFilesForAccount returns a list of OCFile`() {
+        every {
+            localFileDataSource.getDownloadedFilesForAccount(OC_ACCOUNT_NAME)
+        } returns listOf(OC_FILE)
+
+        val result = ocFileRepository.getDownloadedFilesForAccount(OC_ACCOUNT_NAME)
+
+        assertEquals(listOf(OC_FILE), result)
+
+        verify(exactly = 1) {
+            localFileDataSource.getDownloadedFilesForAccount(OC_ACCOUNT_NAME)
+        }
+    }
+
+    @Test
+    fun `getDownloadedFilesForAccount returns an empty list when datasource returns an empty list`() {
+        every { localFileDataSource.getDownloadedFilesForAccount(OC_ACCOUNT_NAME) } returns emptyList()
+
+        val result = ocFileRepository.getDownloadedFilesForAccount(OC_ACCOUNT_NAME)
+
+        assertEquals(emptyList<OCFile>(), result)
+
+        verify(exactly = 1) { localFileDataSource.getDownloadedFilesForAccount(OC_ACCOUNT_NAME) }
+    }
```

Comment:
This is not needed, the result is what the datasource returns, we're not giving it a special treatment in-between

Reply:
Ok, deleted